### ISSUE TITLE
Valid Pairs File is Empty

### ISSUE THREAD

**Issue by chadi-elfarran** [nservant/HiC-Pro#243]
Hi nservant,

Thank you for creating this wonderful tool.

I am currently trying to analyze HiChIP samples. My HiC-Pro run crashes at the "Run quality checks for all samples" step with the following error:
"/home/jllab/Apps/HiC-Pro-master/bin/../scripts//Makefile:178: recipe for target 'hic_qc' failed
make: *** [hic_qc] Error 1"

The hierarchy of the folders is as per the manual:
  + Samples
     + Oct4_HiChIP_B1T1
       ++ SRR3928687_R1.fastq.gz
       ++ SRR3928687_R2.fastq.gz
     + Oct4_HiChIP_B1T2
       ++ SRR3928688_R1.fastq.gz
       ++ SRR3928688_R2.fastq.gz

I have also removed the /1 and /2 from the end of the read names (Public datasets downloaded from ENA).

I also ensured that the chromosome names are the same in the annotation files and genome index.

By inspecting the log folder I found the following issues:

--All pairs were found to be Dumped_pairs. No valid interaction detected at all.

--In the mapping_combine.log the samtools sort shows the following errors:
"[E::hts_open_format] Failed to open file
samtools sort: failed to create temporary file "tmp/SRR3928687_R2_mm9.0000.bam": File exists"
This happens for some of the temporary files for both R1 and R2. When I look in the tmp inside the Output directory, I can see all the temporary files from R2 sill exist. I don't know if this is indicating any issue.

--The plot_mapping_portion.Rout file shows: 
"Error: length(allmapstat_r1) > 0 && length(allmapstat_r2) > 0 is not TRUE
Execution halted"

I am using Samtools Version 1.9 and HiC-Pro 2.11.1 in Ubuntu 16.04.

Thank you for the help.

Best Regards,

Chadi

**Comment by nservant** [comment]
Hi Chadi,
It seems that samtools sort failed at some points.
Not sure why, but I think that you will need to rerun the pipeline ...
N

**Comment by chadi-elfarran** [comment]
I have re-run the pipeline. Unfortunately same issue.

I have also deleted and reinstalled HiC-Pro. Similar error. All reads are considered dumped pairs.

I really don't know what is the issue. Any help or suggestion will be truly appreciated.

Best Regards,

Chadi

**Comment by nservant** [comment]
I think that this is not linked to your installation.
Can you try to run the test dataset please ?

**Comment by chadi-elfarran** [comment]
Thank you for the suggestion.

The test dataset runs smoothly without errors.

I identified the problem which turned out to be in the configuration file. I have changed the PAIR1_EXT and PAIR2_EXT to _1.fastq.gz and _2.fastq.gz respectively.  The ".fastq.gz" in the configuration file caused all the problem. All I had to do was to make it "_1" and "_2".

Thank you for the help and for for this wonderful tool.

Best Regards,

Chadi
